### PR TITLE
Update tests to run under an AWS cluster

### PR DESCRIPTION
**Pull request by mgummelt**

### CODE REVIEW

**Comment by jsancio** [comment]
FYI, @mgummelt and I talked offline. We are going to keep this PR around until CCM becomes more reliable.

File: cli/tests/integrations/test_service.py [hunk 1]
```diff
@@ -146,17 +146,19 @@ def test_log_marathon_config():
...
-        'dcos service log marathon ' +
-        '--ssh-config-file=tests/data/service/ssh_config')
-
-    assert len(stdout.decode('utf-8').split('\n')) > 10
-
-    assert b"Running `" in stderr
-    num_lines = len(stderr.decode('utf-8').split('\n'))
-    assert ((num_lines == 2) or
-            (num_lines == 3 and b'Warning: Permanently added' in stderr))
+# Now that we test against an AWS cluster, this test is blocked on
```
Comment (jsancio) [inline cli/tests/integrations/test_service.py]: We shouldn't comment out code. Let's remove it or skip it.

**Comment by mgummelt** [comment]
@jsancio: This is now passing on my custom TC build configuration.  I'll leave it unmerged so we can monitor the build, but it would be nice to get a LGTM on this now so it's ready to go.

**Comment by jsancio** [comment]
Actually, @mgummelt. Can we commit this without the Jenkins bits for now? This in an strict improvement on our tests, no? We can create another PR for the Jenkins side of things.

**Comment by mgummelt** [comment]
Still fails ~5% of the time the last time I checked.  Been waiting on a CCM
bug for too long now.

**Comment by mgummelt** [comment]
@jsancio 

The blocking CCM bug is supposedly fixed.  Please review.

**Comment by tamarrow** [comment]
this was forked and updated in [pr 349](https://github.com/mesosphere/dcos-cli/pull/395) so I'm closing this pr.

**Pull request closed without merging**